spec/components/timeline/Timeline.spec.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
```

Comment:
Look at the documentation in `contributing.md`:

`- Even if Enzyme exists we'll use from now on only Testing Library`

Reply:
Done